### PR TITLE
global: initial REST validation API

### PR DESCRIPTION
**Pull request by mvesper**
- Adds _dry_run_ key to the webhooks payload to validate the action.

Signed-off-by: Martin Vesper martin.vesper@cern.ch

### CODE REVIEW

**Comment by jirikuncar** [comment]
What about adding `dry-run` or `only-validate` option to the endpoints?

## 

Alternatively, I would change the name from `Test*` to `Validate*` and use inheritance instead of specifying the validator twice.

``` pass
class ValidatorFoo(ValidatorBase, Foo):
    pass
```

File: invenio_circulation/receivers.py [hunk 1]
```diff
@@ -54,6 +54,9 @@ def run(self, event):
         else:
             data = event.payload
 
+        if 'dry_run' in event.payload and event.payload['dry_run']:
```
Comment (jirikuncar) [inline invenio_circulation/receivers.py]: `if event.payload.get('dry_run'):`

**Comment by jirikuncar** [comment]
LGTM

**Comment by mvesper** [comment]
@jirikuncar I saw that you 'thumbed-up' the commit message. I changed it though. Is it still valid? Should be fine otherwise :)

File: setup.py [hunk 2]
```diff
@@ -62,7 +62,7 @@
 install_requires = [
     'Flask-BabelEx>=0.9.2',
     'Flask>=0.11.1',
-    'invenio-assets>=1.0.0b1',
+    'invenio-assets>=1.0.0a2',
```
Comment (jirikuncar) [inline setup.py]: b2

**Pull request merged**